crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/load.rs

```diff
@@ -0,0 +1,36 @@
+use crate::state::BlockifierState;
+use cairo_felt::Felt252;
+use conversions::{FromConv, IntoConv};
+use num_traits::Pow;
+use starknet::core::types::FieldElement;
+use starknet_api::core::{ContractAddress, PatriciaKey};
+use starknet_api::hash::StarkHash;
+use starknet_api::state::StorageKey;
+
+pub fn load(
+    blockifier_state: &mut BlockifierState,
+    target: ContractAddress,
+    storage_address: &Felt252,
+    size: &Felt252,
```

Comment:
I think this function and its arguments should be described. I found it difficult to understand

Reply:
After reading the tests I've understand it, but please add some comments 🙏